### ISSUE TITLE
quick help app not accessible with nvda

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#13742]
### Steps to reproduce:
in the start menu type quick assistance.
press enter.
You should receive an offer to download the new microsoft store quick help app.
Download the new app from the microsoft store.
At the end of the installation press enter on open app.
### Actual behavior:
when walking with tab or arrows nvda is muted.
when using object navigation when we enter within quick assistance nvda only finds the bar to maximize and minimize the window.
### Expected behavior:
enable navigation with tabs or arrows through the app.
#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
nvda.exe, NVDA alpha-25478,841925b7

#### Windows version:
10 19044.1706
#### Name and version of other software in use when reproducing the issue:
QuickAssist.exe, MicrosoftCorporationII.QuickAssist 2.0.6.0

#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes
log
[assistencia.txt](https://github.com/nvaccess/nvda/files/8781753/assistencia.txt)

**Comment by seanbudd** [comment]
Can you test if this works with Narrator?
This is very likely a problem with the implementation of the application/website, as a starting point please submit feedback to the authors to let them know about the problem.

**Comment by fernando-jose-silva** [comment]
using narrator, the app It is accessible only in scan mode.
 on nvda I was able to access using the flat review.  but this requires the user to juggle a lot, assigning the commands to flat review unassigned by default, putting the system focus on the flat review focus, and using the special command to trigger the desired object, study in the narrator, requires much less learning , it only requires putting the screen reader in scanning mode and with enter activate the desired one.  i think an invisible window is taking focus, narrator announces opening easy access, in nvda when using flat review and throwing system focus where flat review is, app is accessible using tab and arrows, it would be nice if some nvda developer could see if there is anything that can be done, anyway, how can we complain to microsoft of this problem?

**Comment by seanbudd** [comment]
I found this link by searching for quick help support: https://support.quickhelp.com/support/tickets/new
